examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,532 @@
+{
```

Comment:
<p>Currently this function is not meant to be used by users. If we want that, let's add it to the <code>__init__.py</code> ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>